thirdparty/download-thirdparty.sh

```diff
@@ -317,7 +305,8 @@ echo "Finished patching $LIBRDKAFKA_SOURCE"
 cd $TP_SOURCE_DIR/$AWS_SDK_SOURCE
 if [ ! -f $PATCHED_MARK ]; then
     if [ $AWS_SDK_SOURCE == "aws-sdk-cpp-1.9.211" ]; then
-         curl -L https://doris-thirdparty-repo.bj.bcebos.com/thirdparty/aws-crt-cpp-1.9.211.tar.gz | tar -zx 
```

Comment:
Unify to `wget` tool in `download-thirdparty.sh`, so that user don't need to install 2 tools.